sample/src/main/java/io/getstream/chat/example/utils/UserStorage.java

```diff
@@ -0,0 +1,84 @@
+package io.getstream.chat.example.utils;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import androidx.annotation.Nullable;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.util.List;
+
+public class UserStorage {
```

Comment:
There is no need to make such kind of classes with all static methods. It's ok for utilities where all static methods are independent and eventually might be tested independently.

But this one defines interconnected methods with common business logic, so it should be hidden behind interface that we can replace and test it:
```java
interface UserStorage {
  UserConfig getCurrentUser();
  //... other methods
}
```
```java
class StreamUserStorage implements UserStorage {
  public static UserStorage get() { return instance; }
  @Override UserConfig getCurrentUser() {}
} 
```